src/tabnineChatWidget/handlers/navigateToLocation.ts

```diff
@@ -0,0 +1,38 @@
+import * as vscode from "vscode";
+
+export interface NavigateToLocationPayload {
+  path: string;
+  range: {
+    startLine: number;
+    endLine: number;
+  };
+}
+export function navigateToLocation(payload: NavigateToLocationPayload): void {
+  void doNavigate(payload);
```

Comment:
why do you need the  wrapping function?

Reply:
to have multiple awaits and `void` them outside, otherwise i'd have to use `.then`s

Reply:
And why do you need to void on them ?

Reply:
oh, im just now seeing that the chat API registry supports async handlers as well, will change